### PR TITLE
💄 Redesign newsletter subscribe form

### LINKED ISSUE #707: 💄Newsletter Page- styling
URL: https://ssw.com.au/newsletters

As per my conversation with @wicksipedia and @bradystroud, our current newsletters' design is not as per modern standards therefore we need to do the following changes to Newsletters component.

```[tasklist]
### Tasks
- [ ] Get designer feedback
- [ ] Newsletters - remove button from the component name
- [ ] Newsletters - Restyle based on Training page
- [ ] Add hero image (Testimonial style)
```

### PR DESCRIPTION
**Pull request by KristenHu**
Fixes #707 - Reskin newsletter page as per Ken's design

Affected routes:

-  `/newsletter`

Add done video, screenshots
image

### CODE REVIEW

**Review by bradystroud — CHANGES_REQUESTED** [review]
image
Dark text on dark background doesnt work

File: components/blocks/subscribe.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { Template } from "tinacms";
+import { Container } from "../util/container";
+import { SubNewsLetters } from "./subNewsLetters";
+
+export const Subscribe = () => {
+  return (
+    <div
+      // eslint-disable-next-line tailwindcss/no-arbitrary-value
+      className={
+        "flex h-80 w-full items-center bg-[url('/images/thumbs/subscribeBackground.png')] bg-cover bg-center bg-no-repeat text-center font-light after:absolute"
```
Comment (bradystroud) [inline components/blocks/subscribe.tsx]:
```suggestion
      className={
        "flex h-80 w-full items-center bg-subscribe-background bg-cover bg-center bg-no-repeat text-center font-light after:absolute"
```

if you add the image to the Tailwind config file, you can avoid disabling this warning 

We have done this for a bunch of other images:
https://github.com/SSWConsulting/SSW.Website/blob/3bc101637f3d268775f5c26ee502ddd015ac7575/tailwind.config.js#L374-L383

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: components/blocks/subscribe.tsx (hunk 1 above)
Comment (KristenHu) [reply components/blocks/subscribe.tsx]: **Done** - Thanks for the advice😃

File: components/blocks/subNewsLetters.tsx [hunk 2]
```diff
@@ -119,10 +119,10 @@ export const SubNewsLetters = ({
   return (
     <div className="flex flex-col items-center justify-center">
       <h2
-        className="mb-2 mt-6 text-center font-helvetica text-4xl font-medium text-sswBlack"
+        className="mb-2 mt-6 text-center font-helvetica text-4xl font-medium text-white"
         dangerouslySetInnerHTML={{ __html: headerText }}
       ></h2>
-      <div className="mb-5 text-center text-base text-sswBlack">
+      <div className="mb-5 text-center text-base text-white">
```
Comment (bradystroud) [inline components/blocks/subNewsLetters.tsx]: image

We need to conditionally change the text color depending on the background color

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: components/blocks/subNewsLetters.tsx (hunk 2 above)
Comment (bradystroud) [reply components/blocks/subNewsLetters.tsx]: You can find this component here https://app-sswwebsite-9eb3-pr-860.azurewebsites.net/?liveStream=1

Comment (KristenHu) [reply components/blocks/subNewsLetters.tsx]: **Done**🙌
image

**Review by bradystroud — APPROVED** [review]

**Pull request merged**